Taskfile.yml

```diff
@@ -161,6 +161,11 @@ tasks:
         vars:
           CLI_ARGS: '--load {{.CLI_ARGS}}'
 
+  build:charts:
+    desc: "Run helm dep build for all charts"
```

Comment:
NIT: It's not all chart only redpanda, but I think this is the place where we should add any additional helm dep invocation.